code/src/components/TodoNavbar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+
+import '../css/TodoNavbar.css'
+import menu from '../assets/menu.png'
+import loupe from '../assets/loupe.png'
+import bell from '../assets/bell.png'
+
+const TodoNavbar = () => {
+    return (
+        <nav className="navbar-wrapper">
+            <div className="menu-icon-container">
+                <img
+                    src={menu}
+                    alt="menu icon"
+                />
+            </div>
+            <div class="other-icon-container">
+                <img 
+                    src={loupe}
+                    alt="search icon"
```

Comment:
maybe another class name for the div that describes the icons more could be easier is someone else is reading your code :) Maybe something including notifications and search or tool-icons or something :)